packages/scss/src/components/_label.scss

```diff
@@ -30,3 +30,16 @@
 		}
 	}
 }
+
+
+.label {
+	&.mod-number {
+		border-radius: 50%;
```

Comment:
Border radius should be equal to height as we are aiming for a pill not an ellipse. 
I think height/line-height/border-radius/min-width should be equal to sizes.standard.line-height